activesupport/lib/active_support/core_ext/string/inflections.rb

```diff
@@ -164,7 +164,7 @@ def deconstantize
   #
   #   <%= link_to(@person.name, person_path) %>
   #   # => <a href="/person/1-donald-e-knuth">Donald E. Knuth</a>
-  def parameterize(sep = '-')
+  def parameterize(sep = '-'.freeze)
```

Comment:
True, if you assign the frozen at trings to constants up front, you can re-use the same instance all over the place. Plus, they won't be GC'ed.